gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```

Comment:
Can this file be ignored and simply generated in the GH workflow? I'm trying to be cautious about gradle things so it's clear it's not used aside from releasing to Maven Central.

Reply:
This was a good point, the `gradle/gradle-build-action` action has the ability to prepare a Gradle installation.
It is good practice **while developing** to use the Gradle wrapper to maintain a consistent Gradle version for all users, however we are not using this component during development, only publishing, and so the benefits are reduced significantly.
As such, I have removed the Gradle wrapper and tested that `gradle/gradle-build-actions` provisioned Gradle for us.